src/components/ProjectList/listOfProjects.js

```diff
@@ -528,6 +528,13 @@ const projectList = [
     githubLink: 'https://github.com/roc-project/roc/contribute',
     description: 'A toolkit for real-time audio streaming over the network',
     tags: ['C++', 'Audio', 'Streaming', 'Networking', 'Cross-Platform', 'Linux', 'MacOS', 'Windows']
+  },
+   {
+    name: 'Conda',
+    imageSrc: 'https://conda.io/en/latest/_images/conda_logo.svg',
+    githubLink: 'https://github.com/conda',
+    description: 'Conda is an open source package management system and environment management system that runs on Windows, macOS and Linux. Conda quickly installs, runs and updates packages and their dependencies. Conda easily creates, saves, loads and switches between environments on your local computer. It was created for Python programs, but it can package and distribute software for any language.',
+    tags: ['ML', 'Python', 'Builds','package manager']
```

Comment:
Capitalise `package manager` to `Package Manager`

Reply:
Done